### DISCUSSION TITLE
API for editing page

### DISCUSSION THREAD

**Discussion by soke1556** [requarks/wiki#3471] [Help / Questions]
**Question**
Is it possible to edit a page (MD) via API or any other automatic method?
I need to update a page based on the content of another file that updates regularly.

**Host Info (please complete the following information):**
OS: Ubuntu (bionic) 18.04 x64
Node.js: 12.13.1
Wiki.js version: 2.5.144
Database engine: PostgreSQL 10.14

**Comment by Igorgro** [comment]
Yes, you can. See https://docs.requarks.io/en/dev/api and https://docs.requarks.io/graphql

**Comment by soke1556** [comment]
Sorry, but all I see there is reading. Can you, please, provide a sample o a hint for updating a page? Thanks in advance.
